docs/data/flink_connectors.yml

```diff
@@ -50,6 +50,13 @@ hbase:
   source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-hbase-4.0.0/flink-connector-hbase-4.0.0-src.tgz.sha512"
   compatibility: ["1.18.x", "1.19.x"]
 
+hive:
+  name: "Apache Flink Hive Connector 3.0.0"
+  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-hive-3.0.0/flink-connector-hive-3.0.0-src.tgz"
+  source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-hive-3.0.0/flink-connector-hive-3.0.0-src.tgz.asc"
+  source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-hive-3.0.0/flink-connector-hive-3.0.0-src.tgz.sha512"
+  compatibility: ["1.18.x"]
```

Comment:
Should be 1.20.x now?

Reply:
compatibility: ["1.18.x", "1.19.x", "1.20.x"]

Reply:
But as this is the first external release of hive connector, the hive connector for flink 1.19.x and 1.20.x has been released together with flink 1.20 ,I agree that 1.20.x is enough here.

Reply:
yep, thanks for the feedback,
this is from RC1 
I forgot to update it after RC2 created

updated now